tasks/IsTheRefrigeratorRunning.tex

```diff
@@ -0,0 +1,37 @@
+\section{Is the Refrigerator Running [Housekeeper]}
+The people who live in the arena are so sloppy! The robot finds minor faults in the state of the arena and rectifies them. This is an advanced version of Clean Up for Stage II.
+
+% \subsection{Focus}
+% This test focuses on object perception, manipulation, and planning.
+
+\subsection{Main Goal}
+The robot wanders around the apartment looking for things that are wrong (we will provide a list of potentially-wrong things), and then fixes them. Examples: The refrigerator door is open, the pillows from the bed are on the floor, the television is on with nobody watching.
```

Comment:
We must provide a comprehensive list of stuff that might be wrong, and then clamp the e.g. 5 things the robot must look for (we mess up 7). As is, the main goal is so fuzzy I wouldn't know how to code it.